cypress/e2e/room_board/copy_room_board/copyRoomBoardWithFilesInFolder.feature

```diff
@@ -0,0 +1,34 @@
+@regression_test
+@stable_test
+Feature: Create, edit and delete folder in board, including file handling inside the folder (in progress)
+
+As a teacher I want to copy a room board with files in file folder
```

Comment:
here may be a full user story is better. As a teccher............... , so that I....... .